### ISSUE TITLE
Move notebooks etc to github

### ISSUE THREAD

**Issue by jph00** [fastai/dlcert1#6]
Need to leave them where they are too since they're referred to in the lessons

**Comment by racheltho** [comment]
I moved the notebooks, but not the zip files.  Also, still need to make the following changes to the redux notebook:
"Some thoughts:
- Rather than looping and saving each epoch, could you simply set nb_epochs to the number you want, and use the callbacks param to handle the weight saving?
- "If the probability is >= .5, Kaggle considers that a dog, otherwise it's a cat" isn't really correct. Kaggle simply uses the log-likelihood (cross-entropy) loss function that is on its evaluation page
- "Unfortunately Kaggle doesn't support prediction values of 0 or 1" is correct. It's the the loss function is infinite in these cases, if you submit a wrong prediction (which makes sense, if you think about it). And the softmax activation doesn't try to create an accurate probability. So you need to post-process the results of the softmax to create a more reasonable probability, to get a good score
